suite/src/__tests__/fast/model-correctness.test.ts

```diff
@@ -0,0 +1,57 @@
+import { describe, test, beforeAll, expect } from '@jest/globals'
+import { newCeramic } from '../../utils/ceramicHelpers.js'
+import { createDid, splitDidsToList } from '../../utils/didHelper.js'
+import { StreamID } from '@ceramicnetwork/streamid'
+import {Model} from '@ceramicnetwork/stream-model'
+import {ModelInstanceDocument} from '@ceramicnetwork/stream-model-instance'
+import { newModel, basicModelDocumentContent } from '../../models/modelConstants'
+import { CeramicClient } from '@ceramicnetwork/http-client'
+import { CommonTestUtils as TestUtils } from '@ceramicnetwork/common-test-utils'
+
+const ComposeDbUrls = String(process.env.COMPOSEDB_URLS).split(',')
+// TODO : Update to the modelId created manually on all envs, if id cannot be the same then create an array for different envs  
```

Comment:
What does the current model ID in the code (`kjzl6hvfrbw6c59m8k5l50gqmsduszfazp25wakaep3nsiyieff7dbdvpgb0ypf`) correspond to?

Reply:
We want to replace this with the model id that we get after creating a model for testing on our durable infras.